### PR TITLE
Merge Dev into master

### PR DESCRIPTION
**Pull request by Yttruire**
Essentially:
- Navigation bar (w/ navigation to home, settings, login/sign up/log out)
- Login/Signup popups
- General reorganisation of most user authentication components/folders
- Misc styling

### CODE REVIEW

File: frontend/src/Components/pages/dashboard/index.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import NavigationBar from "../../navigationbar";
+import { Grid } from "@material-ui/core";
+
+const Dashboard = () => {
+  return (
+    <Grid container direction="column" justify="space-between" alignItems="stretch" spacing={2}>
+      <Grid item>
+        <NavigationBar />
```
Comment (junwei26) [inline frontend/src/Components/pages/dashboard/index.js]: We should probably shift the nav bar to the App.js level so it doesnt render a new one for each page

File: frontend/src/Components/pages/landingpage/index.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import NavigationBar from "../../navigationbar";
+import { Grid } from "@material-ui/core";
+
+const LandingPage = () => {
+  return (
+    <Grid container direction="column" justify="space-between" alignItems="stretch" spacing={2}>
+      <Grid item>
+        <NavigationBar />
```
Comment (junwei26) [inline frontend/src/Components/pages/landingpage/index.js]: Same here

File: frontend/src/Components/pages/settingspage/index.js [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      setName(newName);
+      setPhotoUrl(newPhotoUrl);
+      setEmail(email);
+    };
+
+    return (
+      <>
+        <Grid container direction="column" justify="space-between" alignItems="stretch" spacing={2}>
+          <Grid item>
+            <NavigationBar />
```
Comment (junwei26) [inline frontend/src/Components/pages/settingspage/index.js]: Same here

**Review by junwei26 — CHANGES_REQUESTED** [review]

File: frontend/src/Components/pages/dashboard/index.js (hunk 1 above)
Comment (Yttruire) [reply frontend/src/Components/pages/dashboard/index.js]: This and the other two comments should be resolved in this commit https://github.com/junwei26/remiNdUS/pull/1/commits/fd0890f238843b571d600f8ca589655a002668e5
This is the first commit right after this PR.

File: frontend/src/Components/pages/landingpage/index.js (hunk 2 above)
Comment (Yttruire) [reply frontend/src/Components/pages/landingpage/index.js]: Same as above reply

File: frontend/src/Components/pages/settingspage/index.js (hunk 3 above)
Comment (Yttruire) [reply frontend/src/Components/pages/settingspage/index.js]: Same as above reply

**Pull request merged**